node/node.go

```diff
@@ -1,7 +1,5 @@
 package node
 
 type Node struct {
-	CPUs   int    `json:"cpus" yaml:"cpus"`
-	ID     string `json:"id" yaml:"id"`
-	Memory string `json:"memory" yaml:"memory"`
+	ID string `json:"id" yaml:"id"`
```

Comment:
We already create these IDs, so there should no more work be required. See https://github.com/giantswarm/cluster-service/blob/b7cd6a88dce33457642a753b457cafb4879f00aa/service/cluster/creator/service.go#L184.